### ISSUE TITLE
Need packaging tests for non-Windows plugins

### ISSUE THREAD

**Issue by csciguy8** [CesiumGS/cesium-unreal#1430]
We'd like to catch potential user issues on other platforms too.

**Comment by csciguy8** [comment]
Could fix issues like this before they get released,
https://github.com/CesiumGS/cesium-unreal/issues/1427

**Comment by kring** [comment]
Indeed! That particular issue is interesting because the problem did get flagged during the plugin build, but it was only a warning and we never saw it. From [the v2.5.0 build log](https://github.com/CesiumGS/cesium-unreal/actions/runs/8910952102/job/24471261518):

> Library '/Users/runner/work/cesium-unreal/cesium-unreal/packages/CesiumForUnreal/HostProject/Plugins/CesiumForUnreal/Source/CesiumRuntime/../ThirdParty/lib/iOS-xarm64/libktx.a' was not resolvable to a file when used in Module 'CesiumRuntime', assuming it is a filename and will search library paths for it. This is slow and dependency checking will not work for it. Please update reference to be fully qualified alternatively use PublicSystemLibraryPaths if you do intended to use this slow path to suppress this warning.

**Comment by yohimik** [comment]
A useful first layer before full non-Windows packaging tests would be an explicit inventory check of the packaged plugin's native libraries. For the `libktx.a` case above, the producer build emitted a warning, but a missing required file for a declared target should fail the packaging gate.

The [current Windows package test](https://github.com/CesiumGS/cesium-unreal/blob/8e45cd9f1be83da72edba2bf32558ffa187831ca/.github/workflows/testPackageOnWindows.yml) already downloads the built plugin artifact into `TestsProject`, which is the right boundary to extend: test the distributed package rather than a source checkout that might supply missing files. For each supported Apple/Linux target, check the expected native library set, then package a minimal consumer using that exact artifact and a pinned UE version.

I checked [v2.29.1](https://github.com/CesiumGS/cesium-unreal/releases/tag/v2.29.1): it has source-only and packaged ZIPs for UE 5.6, 5.7, and 5.8. I have not rebuilt them or inspected their full payloads, so I am not claiming that the historical iOS omission remains in this release.

Disclosure: I maintain [dispat](https://github.com/yohimik/dispat), which can keep shared release steps in referenced configuration with explicit Windows/Linux shell overrides. A native-libraries → plugin-package → consumer-test graph is a relevant use case. It would still need the real platform runners, Unreal toolchains, and the inventory/packaging tests above; orchestration does not turn a successful host build into evidence for every target platform.
